### DISCUSSION TITLE
About the bug of tool request-sender

### DISCUSSION THREAD

**Discussion by maoyuliuxin** [he3-app/he3-feedback#273] [Bug Report]
### Version
request-sender 1.0.8

### Environment
Mac v1.3.23

### Steps to reproduce
http request sender使用时切换其他tab，请求丢失

### What is expected?
可以保存历史请求，有些请求需要重复使用

**Comment by lyzhang1999** [comment]
感谢反馈，我们已经记录了 issues：https://github.com/he3-app/he3-feedback/issues/276
